fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -1240,8 +1243,7 @@ Reference<ILogSystem::IPeekCursor> TagPartitionedLogSystem::peekLogRouter(UID db
 				}
 			}
 			int bestSet = bestPrimarySet;
-			if (SERVER_KNOBS->LOG_ROUTER_PEEK_FROM_SATELLITES_PREFERRED && bestSatelliteSet != -1 &&
-			    tLogs[bestSatelliteSet]->tLogVersion >= TLogVersion::V4) {
+			if (useSatellite && bestSatelliteSet != -1 && tLogs[bestSatelliteSet]->tLogVersion >= TLogVersion::V4) {
```

Comment:
I am a bit confused about this... I suppose this is a back port but in the main branch I still see this old code.

Reply:
The main hasn't been updated with https://github.com/apple/foundationdb/pull/9601 yet, which has this change.